packages/vscode-extension/src/common/utils.ts

```diff
@@ -18,7 +18,7 @@ export interface UtilsInterface {
 
   saveMultimedia(multimediaData: MultimediaData): Promise<boolean>;
 
-  movePanelToNewWindow(): Promise<void>;
+  movePanelTo(location: "new-window" | "editor-tab" | "side-panel"): Promise<void>;
```

Comment:
Since this union type is used in multiple places, I'd prefer
```suggestion
  movePanelTo(location: IDEPanelLocation): Promise<void>;
```
With a `type IDEPanelLocation = "new-window" | "editor-tab" | "side-panel"` alias somewhere else, or changing the existing `PanelLocation` union to include the new `"new-window"` value and using that.